appeals/web/src/server/views/appeals/documents/document-upload.njk

```diff
@@ -7,7 +7,13 @@
 	{{ pageTitle | default('GOV.UK') }} – {{ serviceName }}
 {% endblock %}
 
-{%- set pageHeading = 'Upload files' -%}
+{%- block pageHeading -%}
+	{%- if pageHeadingText -%}
+		<span class="govuk-caption-l govuk-!-margin-bottom-3">{{pageHeadingText}}</span>
```

Comment:
In this case, would the page have no `h1`? If so, I think this should be an `h1` with appropriate classes to style it if it needs to visually appear as a caption

Reply:
Done